Adding icons for extensions (16px & 24px)

zleach opened the pull request.

zleach committed:
Adding icons for extensions (16px & 24px)

jessicalc commented:
Hey! Can you do Step #2 of the instructions in here: https://github.com/optimizely/oui-icons#adding-and-editing-icons 

i.e. update the Changelog file under [Unreleased]
Here's an example: https://github.com/optimizely/oui-icons/commit/635ea63eb3735dcfd70ea149ae9737f0ebfa6846 

Thanks!

jessicalc commented:
talked offline with zach about this, i'll update the changelog

Commit:
Update changelog, update names to be plural for usability (we typically refer to extensions in plural
form, and not singular, so this should help prevent errors from happening if people are manually
typing in the icon names)
